pkg/workspace/workspace.go

```diff
@@ -11,6 +11,11 @@ import (
 	"github.com/warptools/warpforge/wfapi"
 )
 
+type codedError interface {
```

Comment:
Shouldn't be necessary.  There's package scope `serum.Code` func that'll do the relevant job.

Reply:
Ah nice.